app/inputs/price_input.rb

```diff
@@ -16,7 +16,7 @@ def to_html
   end
 
   def input_html_options
-    super.merge({ value: value, min: 0, step: 0.01 })
+    super.merge({ value: value, step: 0.001 })
```

Comment:
why this change?

Reply:
_fixes price input so that tests pass_

According to tests, this is what price input should accept.